### DISCUSSION TITLE
Is it possible to use Gitlab for building and deploying CI/CD for ADF?

### DISCUSSION THREAD

**Discussion by NowinskiK** [Azure-Player/azure.datafactory.tools#100] [Q&A] [answered]
[Sushanta Meher asked on my YT channel](https://www.youtube.com/watch?v=E0yXXegenTM&lc=UgxgjUv3qW8YaR6gttB4AaABAg)

**Comment by NowinskiK** [comment, marked as answer]
Yes, absolutely you can. However, you would have to use the PowerShell module as a replacement for the DevOps task available in Microsoft Marketplace.

**Reply by sinapali** [reply]
Thanks @NowinskiK. It would be great if you could help with an example, or may be create a YouTube video tutorial on the same.
